lib/broccoli/ember-app.js

```diff
@@ -366,6 +370,22 @@ class EmberApp {
 
     this.options = defaultsDeep(options, detectedDefaultOptions, DEFAULT_CONFIG);
 
+    if (this.options.addons.blacklist) {
```

Comment:
Should I set `this.options.addons.blacklist` and `this.options.addons.whitelist` when not defined to respectively `this.options.addons.exclude` and `this.options.addons.include`? Just in case e.g. a project starts using `include` / `exclude` and one of its addons uses `blacklist` / `whitelist` to perform a build-time check.

Reply:
Let's add a getter and issue a warning if someone does touch `this.options.addons.blacklist` etc